flixel/input/gamepad/id/MFiID.hx

```diff
@@ -25,8 +25,8 @@ class MFiID
 	public static inline var DPAD_LEFT:Int = 19;
 	public static inline var DPAD_RIGHT:Int = 20;
 	
-	public static var LEFT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(0, 1);
-	public static var RIGHT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(2, 3);
+	public static var LEFT_ANALOG_STICK (default, null) = new FlxGamepadAnalogStick(0, 1, {up:21, down:22, left:23, right:24});
```

Comment:
They are "arbitrary" in that they don't match any physical inputs, but are specifically chosen to not conflict with any existing button ID's. They are basically new virtual digital button inputs that represent the analog stick being pushed past the digital threshold in each direction.